### ISSUE TITLE
Name conflict: MetaInfo overwrites FilePost's new meta property

### ISSUE THREAD

**Issue by claus** [haadcode/ipfs-post#1]
https://github.com/haadcode/ipfs-post/blob/master/src/Post.js#L41

Maybe do something like this?

``` javascript
post.meta = Object.assign(post.meta || {}, new MetaInfo(post.type, size, new Date().getTime(), data.from))
```

**Comment by haadcode** [comment]
Fixed in `0.0.4` with this commit https://github.com/haadcode/ipfs-post/commit/84df95f185cb4e619db6ff0e3b7e1e8f0f57f51f

**Comment by haadcode** [comment]
Thanks for reporting @claus!
